packages/web-components/src/components/ic-dialog/ic-dialog.tsx

```diff
@@ -440,7 +443,10 @@ export class Dialog {
 
     let nextFocusEl = this.getNextFocusEl(this.focusedElementIndex);
 
-    const isHidden = getComputedStyle(nextFocusEl).visibility === "hidden";
+    const isHidden =
+      getComputedStyle(nextFocusEl).visibility === "hidden" ||
+      (nextFocusEl.tagName === this.IC_ACCORDION_GROUP &&
+        nextFocusEl.hasAttribute("single-expansion"));
```

Comment:
The overall fix works as expected but this confused me for accordion. Is the check for the single-expansion so that it skips the see all/hide all button as that is not rendered when a single expansion?

Reply:
Correct. When single expansion, there is no see\hide button so the first accordian element needs to be focussed instead (as the current focussed element is the accordian group which no longer delegates focus)

Reply:
Was trying to figure out why `ic-accordion-group` was registering as an interactive element even though there is no button but I can see it in the slottedInterativeElement array.